README.md

```diff
@@ -51,6 +51,16 @@ cargo generate gh:hydro-project/hydroflow-template
 and you will get a well-formed Hydroflow/Rust project to use as a starting point. It provides a simple Echo Server and Client, and advice
 for adapting it to other uses.
 
+### Enable Ligature Support In IDE
```

Comment:
In The IDE? In Your IDE?

Reply:
Going with "Enable IDE Support For Ligatures"